server/src/main/java/io/littlehorse/common/LHServerConfig.java

```diff
@@ -659,6 +660,7 @@ public long getTimerMemtableSize() {
     public Properties getKafkaProducerConfig(String component) {
         Properties conf = new Properties();
         conf.put("client.id", this.getClientId(component));
+        conf.put(CommonClientConfigs.METADATA_RECOVERY_STRATEGY_CONFIG, "rebootstrap");
```

Comment:
I belive it is possible to use a constant "rebootstrap" instead of the string everywhere